coding/using_boost.rst

```diff
@@ -4,24 +4,38 @@ Using Boost
 
 .. _cpp_using_boost:
 
-Certain Boost libraries are recommended; they should be used whenever applicable in preference to any other method of accomplishing the same effect. In general, any library that is tagged with *"Standard .... {something}"* in the `Boost library listing <http://www.boost.org/doc/libs>`_ falls into this category, unless the feature is already supported by the currently-mandated LSST ``g++`` version.
-Among others, this category includes:
+A Boost library shall be used in in preference to any other method of accomplishing the same effect only if:
```

Comment:
Once upon a time we thought that something available in Boost was typically better than any competitor, if for no other reason than we were already depending on Boost.  Perhaps that is no longer true.

Also, my suggested rewrite doesn't have the same implication that Boost is preferable to anything else.